modules/bullet/shape_bullet.h

```diff
@@ -48,6 +49,17 @@ class ShapeBullet;
 class btCollisionShape;
 class ShapeOwnerBullet;
 class btBvhTriangleMeshShape;
+class btCapsuleShape;
+
+class CapsuleShapeY : public btCapsuleShape {
```

Comment:
Why don't use `btCapsuleShape` directly? We don't have multiple capsules oriented differently after all, so why be so specific?

Reply:
Ah, ok, I will fix that if it's not desired.

Reply:
Done!